scripts/data_migrations/2019-02-18-remove-homepage-topic-subtopic-records.sql

```diff
@@ -0,0 +1,5 @@
+UPDATE measure_version SET parent_guid = null, parent_version = null;
```

Comment:
How does this not also need `parent_id` set to NULL?  Would have expected it to choke on that.,

Reply:
Interesting ... I don't know 🤔